src/main/scala/org/hammerlab/guacamole/reference/ContigName.scala

```diff
@@ -0,0 +1,55 @@
+package org.hammerlab.guacamole.reference
+
+/**
+ * Helpers for dealing with contig-name strings.
+ */
+object ContigName {
+
+  // Sort order: chr1, chr2, …, chr22, chrX, chrY, <other>.
```

Comment:
Ah I'm realzing now this PR is actually a bit dated, since we already talked about this contig ordering stuff -- I'll hold off on reviewing any more of it now. Sorry about that!